src/components/target-pane/target-pane.jsx

```diff
@@ -0,0 +1,116 @@
+const React = require('react');
+
+const MediaLibrary = require('../../lib/media-library');
+const VM = require('scratch-vm');
+
+const BackdropLibrary = require('../../containers/backdrop-library.jsx');
+const CostumeLibrary = require('../../containers/costume-library.jsx');
+const SpriteLibrary = require('../../containers/sprite-library.jsx');
+const SpriteSelectorComponent = require('../sprite-selector/sprite-selector.jsx');
+const StageSelector = require('../../containers/stage-selector.jsx');
+
+const styles = require('./target-pane.css');
+
+/*
+ * Pane that contains the sprite selector, sprite info, stage selector,
+ * and the new sprite, costume and backdrop buttons
+ * @param {object} props Props for the component
+ * @returns {React.Component} rendered component
+ */
+const TargetPane = function (props) {
+    const {
+        editingTarget,
+        mediaLibrary,
+        backdropLibraryVisible,
+        costumeLibraryVisible,
+        spriteLibraryVisible,
+        onNewSpriteClick,
+        onNewCostumeClick,
+        onNewBackdropClick,
+        onRequestCloseBackdropLibrary,
+        onRequestCloseCostumeLibrary,
+        onRequestCloseSpriteLibrary,
+        onSelectSprite,
+        stage,
+        sprites,
+        vm,
+        ...componentProps
+    } = props;
+    return (
+        <div
+            className={styles.targetPane}
+            {...componentProps}
+        >
+            <SpriteSelectorComponent
+                selectedId={editingTarget}
+                sprites={sprites}
+                onSelectSprite={onSelectSprite}
+            />
+            <StageSelector
+                backdropCount={stage.costumeCount}
+                id={stage.id}
+                selected={stage.id === editingTarget}
+                url={stage.costume && stage.costume.skin}
+                onSelect={onSelectSprite}
+            />
+            <p className={styles.targetPaneLibraryButtons}>
+                <button onClick={onNewSpriteClick}>New Sprite</button>
```

Comment:
Just a note, maybe soon is the time we might consider working on a localization plan for scratch-gui and scratch-blocks? doing it after the fact with ScratchJr was no fun :)